streaming_providers/seedr/utils.py

```
@@ -37,6 +37,8 @@ async def get_seedr_client(user_data: UserData) -> Seedr:
             if "error" in response:
                 raise ProviderException("Invalid Seedr token", "invalid_token.mp4")
             yield seedr
+    except SyntaxError:
+        raise ProviderException("Invalid Seedr token", "invalid_token.mp4")
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)